### DISCUSSION TITLE
奇忽局05号理事会：NFTs艺术文化事务理事会

### DISCUSSION THREAD

**Discussion by Sleeper2060** [qihuju/qihuju.github.io#25] [General]
根据@Lupingyue 的建议，和最新的一项NFT艺术项目的需要，
现成立奇忽局05号理事会：NFTs艺术文化事务理事会。

该理事会将涉及到推进奇忽局新的NFTs艺术项目的策划，
提供相关资讯和辅助有需要的艺术工作者进行创作，
及一切相关的可以共同协作的NFT计划和展出。

自由加入，自由退出。
有意愿加入或退出请跟帖留言说明。
跟帖后将被为加入或退出这个理事会。
加入后协同处理相关决定和事务。

欢迎提出本理事会规则的增加或修改！！！

成员名单:
@Lupingyue  @MingxuanXie  @yutseng 

@Chemdah --“修格奇斯”的作者
@midai17      -- Facilitator/assistant curator
@Sleeper2060     --策划、创意、技术提供
@Tz-H                   --提供点子创意脑洞以及相关有趣技术实现的检索工作
 @Ungigdu    --提供区块链技术（智能合约/公链/帮助艺术家记录自己的创作）

项目：
1. [“加密奇小怪”NFT项目策划书](https://github.com/qihuju/qihuju.github.io/projects/2)
2. [《loopcraft2020-2021》NFT项目策划书](https://github.com/qihuju/qihuju.github.io/projects/3)
3. [奇忽局宣言 共同协作版 1.0](https://github.com/qihuju/qihuju.github.io/projects/4)

**Comment by Lupingyue** [comment]
> 根据@Lupingyue 的建议，和最新的一项NFT艺术项目的需要，
> 现成立奇忽局05号理事会：NFTs艺术文化事务理事会。
> 
> 该理事会将涉及到推进奇忽局新的NFTs艺术项目的策划，
> 提供相关资讯和辅助有需要的艺术工作者进行创作，
> 及一切相关的可以共同协作的NFT计划和展出。
> 
> 自由加入，自由退出。
> 有意愿加入或退出请跟帖留言说明。
> 跟帖后将被为加入或退出这个理事会。
> 加入后协同处理相关决定和事务。
> 
> 欢迎提出本理事会规则的增加或修改！！！

加入。
期待本理事会的规则的不断迭代。
本理事会的首要任务应该是配合 @Sleeper2060 推进第一个NFTs策展计划，在此过程中也会不断定义参与方的贡献和回报的细节「这个部分 @askender 有很多技术工具的建议」。
期待看到 @Sleeper2060 逐渐释出更多策展计划的理念、玩法和细节。
目前可以确定参加这个策展计划的是：奇小怪的NFT创作。「 链接：1，@Sleeper2060 建立的[GitHub专案入口](https://github.com/qihuju/qihuju.github.io/projects/2) ；2，我建立的讨论专题 #11 」
另外， @MingxuanXie 的《桃花源NFT AR画廊》也可能参与这个计划，具体的意愿和细节等待谢明炫的意见。

**Reply by Ungigdu** [reply]
加入

**Reply by Lupingyue** [reply]
> 加入

欢迎👏👏👏

**Reply by Lupingyue** [reply]
> 加入

你在奇忽局微信群吗？让 @Sleeper2060 把您拉进去。

**Reply by Sleeper2060** [reply]
大家有时间请到奇忽局第一个[NFT project](https://github.com/qihuju/qihuju.github.io/projects/2)
看看，在意见收集栏留言提提意见和建议。
@Lupingyue @MingxuanXie @Ungigdu @yutseng

**Reply by Lupingyue** [reply]
> 大家有时间请到奇忽局第一个[NFT project](https://github.com/qihuju/qihuju.github.io/projects/2)
> 看看，在意见收集栏留言提提意见和建议。
> @Lupingyue @MingxuanXie @Ungigdu @yutseng

好的，这就去留个言。

**Comment by MingxuanXie** [comment]
加入👻

关于线上展览更大的可能性：
在现实中组织一个艺术展需要相当大的人力物力财力，甚至要青年艺术家出上千块来“购买”一个展位，但在线上可以很低的成本来搭建虚拟展览，即便是向艺术家收取一美刀补贴服务器成本也很容易接受。而且现如今数字艺术家越来越多，在线下展览数字作品很多也就是电脑链接个投影仪，跟观众直接在自己电脑上看没本质区别——不过是提供了一个“对话”空间。

只要我们能够把“对话”这部分机制通过数字技术重建出来（比如在clubhouse里组织观展活动，在奇乎局网站上发布展评），并固定下来（一月一展或三月一展），把线上展览定期活动化，甚至提供“线上驻留”，加上接入NFT等收藏机制，坚持做下去想必会很有意义。其影响力其实不会低于一个实体画廊。
e.g 
[ZKM的一年一度线上驻留+展览](https://zkm.de/en/project/web-residencies)  
[Museum of Other Realities](https://www.museumor.com/)

**Reply by Lupingyue** [reply]
> 加入👻
> 
> 人在外面晚點可能再多寫點想法

期待等待⌛️

**Reply by Sleeper2060** [reply]
👏👏

**Reply by MingxuanXie** [reply]
已更新想法

**Reply by Lupingyue** [reply]
> 
> 只要我们能够把“对话”这部分机制通过数字技术重建出来（比如在clubhouse里组织观展活动，在奇乎局网站上发布展评），并固定下来（一月一展或三月一展），把线上展览定期活动化，甚至提供“线上驻留”，加上接入NFT等收藏机制，坚持做下去想必会很有意义。其影响力其实不会低于一个实体画廊。
> e.g
> [ZKM的一年一度线上驻留+展览](https://zkm.de/en/project/web-residencies)
> [Museum of Other Realities](https://www.museumor.com/)

**1，对话机制的数字重建**

> clubhouse组织观展 

> 奇忽局网站上发布展评

奇忽局club接下来除了moderator模式，开始拓展curator模式

**2，固定周期「一月一展或三月一展」，线上展览定期活动化**

**3，线上驻留** 

@MingxuanXie 能提供一些线上驻留的有趣案例吗？

**4，NFT收藏机制** #18 

**5，低成本的服务器补贴**

这几项建议有价值，考虑起来。两个案例链接认真研究一下。

**Reply by yutseng** [reply]
是的。
只是線上展還是要有策展觀念比較好

**Reply by Lupingyue** [reply]
> 是的。
> 只是線上展還是要有策展觀念比較好

嗯哼，赛博策展目前只是古典策展的“辅助”，但是慢慢会“反客为主”，但是以赛博空间为主场的策展需要有“赛博原生”的策展观念、收藏机制「NFT或其它」、批评体系、赛博展示场景、相关的硬件和软件的开发，当然最重要的：伟大的赛博原生艺术家。

**Comment by yutseng** [comment]
加入，我來了

**Reply by yutseng** [reply]
鈺涓

**Reply by Lupingyue** [reply]
> 鈺涓

Wow， 看到鈺涓老師來了，開心。

**Reply by yutseng** [reply]
來了來了，原來我有帳號😄😄

**Reply by Lupingyue** [reply]
> 來了來了，原來我有帳號😄😄

看來是有年頭了，你都忘記了哈哈😄😄

**Reply by Lupingyue** [reply]
> 來了來了，原來我有帳號😄😄

一不小心本理事會已經5個成員了，已經是奇忽局第一大幫派了😂

**Reply by Sleeper2060** [reply]
> > 來了來了，原來我有帳號😄😄
> 
> 一不小心本理事會已經5個成員了，已經是奇忽局第一大幫派了😂

好棒!!👏

**Reply by yutseng** [reply]
是很久一起的帳號了。

> > 來了來了，原來我有帳號😄😄
> 
> 
> 
> 看來是有年頭了，你都忘記了哈哈😄😄

**Comment by Lupingyue** [comment]
除了参与 #11 项目，我也会有一件个人作品参与这次策展计划：《loopcraft2020-2021》
「点击查看“岳路平脑图个展”：https://qihuju.wiki/docs/qihuju-links/yueluping.html 」

这是我在2020-2021疫情期间在Minecraft上创作的一个虚拟博物馆，这将是我第一次呈现它的全貌。
《loopcraft2020-2021》包括：
一幅脑图：相当于这个虚拟博物馆的一张地图
1001张截图，全面呈现这个虚拟博物馆的视觉全景
360张卡牌，根据这个虚拟博物馆的部分内容设计的系列狼人杀卡牌和麻将卡牌

我已经模仿 @Sleeper2060 撰写 #11 project的模式写了一个new project：[《loopcraft2020-2021》NFT计划](https://github.com/qihuju/qihuju.github.io/projects/3)，欢迎大家吐槽，给出改进意见

**Reply by Ungigdu** [reply]
建议在github上新建一个组织，方便管理和搜索

**Reply by Lupingyue** [reply]
> 建议在github上新建一个组织，方便管理和搜索

新建一个组织？是跟奇忽局GitHub社区平行的一个新的GitHub社区吗？

**Reply by MingxuanXie** [reply]
大概指的是github的organisation吧

**Reply by Lupingyue** [reply]
> 大概指的是github的organisation吧

organisation是一个栏目吗？作为萌新用户还不太熟悉。

**Reply by MingxuanXie** [reply]
organisation是高于repository及用户账户的存在，每个人都可以加入及成立一个organisation，organisation可以对单个用户权限进行更详细的限定

![image](https://user-images.githubusercontent.com/31252716/113817427-e52a8100-97a8-11eb-85c7-ee62473f72a7.png)
比如我也是虚幻引擎的组织成员，只有组织成员能够查看修改虚幻引擎源码

目前我们这个repo本来就是Tz新建了一个叫qihuju的账户来成立的，我感觉暂时不用升级成organisation

**Reply by Lupingyue** [reply]
是一定级别自动升级？还是需要交费升级？

**Reply by MingxuanXie** [reply]
有免费的版本 https://github.com/organizations/plan
开源项目应该不需要用到付费版

**Reply by Lupingyue** [reply]
> 有免费的版本 https://github.com/organizations/plan
> 
> 开源项目应该不需要用到付费版

但是目前我们这个阶段其实没有必要升级对吧

**Comment by Sleeper2060** [comment]
@MingxuanXie 提出由他来建立虚拟空间，暂时只能容纳100人。
--100人的空间我觉得已经可以满足现在的需求。
--如果Mingxuan觉得ok，现在可以开始建立这个虚拟空间。
这个空间可以不仅仅是做展示nft作品用，
也可以用作交流空间使用，例如增加奇忽局除了clubhouse的另外一个场所使用。
--其他人想参与建立这个空间可以跟MingxuanXie协商。
--给这个虚拟空间新建了一个[issue](https://github.com/qihuju/qihuju.github.io/issues/31)
有建议或者意见的可以在issue直接留言，也可以直接和MingxuanXie沟通。

**Reply by Lupingyue** [reply]
完美！

**Reply by Lupingyue** [reply]
@Sleeper2060 ：你在project建议Manifesto绑在一个NFT就可以。

那么，Manifesto的形式是什么？
是以图片的形式吗？那么图片里应该是什么内容呢？

**Reply by Sleeper2060** [reply]
Manifesto可以以文字的方式附在nft的说明里。nft有个选择性的玩法，就是购买后才能看到那段文字。这个对我们的Manifesto意义不大，就随嘴这么提一下。哈哈。

**Reply by Lupingyue** [reply]
> Manifesto可以以文字的方式附在nft的说明里。nft有个选择性的玩法，就是购买后才能看到那段文字。这个对我们的Manifesto意义不大，就随嘴这么提一下。哈哈。

哈哈，明白了。

**Comment by midai17** [comment]
我也想作为Facilitator/assistant curator 一起玩儿。我应该已经在微信群里了。

**Reply by Sleeper2060** [reply]
欢迎👏👏

**Reply by Sleeper2060** [reply]
可以去看看我们的第一个nft project，把奇忽局的logo nft化。
在project栏里。

**Reply by Lupingyue** [reply]
> 我也想作为Facilitator/assistant curator 一起玩儿。我应该已经在微信群里了。

歡迎，👏👏
你在微信群叫什麼名字？

**Reply by Lupingyue** [reply]
> 可以去看看我们的第一个nft project，把奇忽局的logo nft化。
> 在project栏里。

嗯嗯，赶快到奇忽局logo NFT化的project去协作⛽️

**Comment by Sleeper2060** [comment]
@Lupingyue 提出“加密奇小怪”NFT项目可以做包含了“玩法”的NFT作品。

--加入玩法会更加有趣，
不过其中涉及到大量的绘画和玩法的设计，
是否可以和一个插画家或者wechat的EMOJI制作者合作，
另起一个项目？如何可以做成表情包，这个就非常好玩，也帮助传播。
--我的project其实有点像哈希面具，
不同的是没有他们那种专业的技术辅助。
要不哈希面具留给我的计划？！😄

**Reply by Lupingyue** [reply]
- 加入“玩法”是 @MingxuanXie 早前的提法，還給出了哈希面具和加密老婆的案例。 
- #3 也提出了奇小怪表情包的建議，很同意另起一個獨立項目
- 很期待看到你的”哈希面具“，好奇會是什麼樣的

**Comment by Chemdah** [comment]
来啦 我是“修格奇斯”的作者Chlor Huang

**Reply by Lupingyue** [reply]
> 来啦 我是“修格奇斯”的作者Chlor Huang

欢迎👏👏，你终于进来了

**Reply by Chemdah** [reply]
最近太忙啦不好意思哎————

**Reply by Lupingyue** [reply]
> 最近太忙啦不好意思哎————

有空到project里看看项目进展

**Reply by Chemdah** [reply]
好的！收到！（大学生标准回信模板.jpg）

**Reply by Sleeper2060** [reply]
👏欢呼。。。。

**Comment by Tz-H** [comment]
申请加入并负责提供点子创意脑洞以及相关有趣技术实现的检索工作

**Reply by Sleeper2060** [reply]
👏👏👏

**Reply by Lupingyue** [reply]
👏👏👏

**Comment by Sleeper2060** [comment]
我们又有第三个Project了。。。
就是模拟区块链的技术来共同合作撰写[宣言](https://github.com/qihuju/qihuju.github.io/projects/4)
虽然是伪NFT，但是意义重大。
@Tz-H 已经起草了非常好的第一版，希望你可以去认领第一块区块。😄

**Reply by Lupingyue** [reply]
这个project意义重要啊

**Reply by Sleeper2060** [reply]
第一块block已建好。🎉好棒。

**Reply by Lupingyue** [reply]
> 第一块block已建好。🎉好棒。

棒呆！

**Reply by Ungigdu** [reply]
可以建真的区块链呀

**Reply by Lupingyue** [reply]
> 可以建真的区块链呀

你来发起一个理事会讨论这个方向吧

**Reply by Ungigdu** [reply]
哈哈，我暂时没有牵头的打算，我这边可以提供技术，无论是智能合约或者是公链都是可以的哈，来帮助艺术家记录自己的创作

**Reply by Lupingyue** [reply]
> 哈哈，我暂时没有牵头的打算，我这边可以提供技术，无论是智能合约或者是公链都是可以的哈，来帮助艺术家记录自己的创作

那就等有谁牵头这个计划再找你请教，感谢这个建议
现在建公链的成本和时间怎么样？

**Reply by Ungigdu** [reply]
现在有波卡的substrate框架，很快就开发完，研发成本也不高。大部分区块链项目的成本在于推广。

**Reply by Lupingyue** [reply]
> 现在有波卡的substrate框架，很快就开发完，研发成本也不高。大部分区块链项目的成本在于推广。

明白，那么根据你的了解：
目前有没有艺术类公链的项目？
之前我曾经构想一个“区块画框”的构想，你觉得可行吗？

**Reply by Ungigdu** [reply]
> > 现在有波卡的substrate框架，很快就开发完，研发成本也不高。大部分区块链项目的成本在于推广。
> 
> 明白，那么根据你的了解：
> 目前有没有艺术类公链的项目？
> 之前我曾经构想一个“区块画框”的构想，你觉得可行吗？

目前没什么出名的公链目标是搞艺术的。NFT项目大部分是在以太坊上运行的，因为NFT很关心资产买卖，以太坊上资产很多，满足这个条件。我不清楚“区块画框”是什么...

**Reply by Lupingyue** [reply]
> 
> 目前没什么出名的公链目标是搞艺术的。NFT项目大部分是在以太坊上运行的，因为NFT很关心资产买卖，以太坊上资产很多，满足这个条件。我不清楚“区块画框”是什么...

1，“区块画框”是一个隐喻，其实就是对“智能合约”的“修辞修饰”，类似“商业模式画布”
2，之所以叫做“区块画框”，是希望吸引“创作类”的人群关注智能合约和区块链
3，“区块画框”的推广模式会采用curating（策展）的玩法，会比一般区块链项目的推广更有趣。类似beeple拍卖事件之于NFT

**Reply by Ungigdu** [reply]
其实大家也不会直接调用智能合约。智能合约有相应的前端页面，前端页面就是各种玩法的说明和入口。只要商业模式成立，都可以做的

**Reply by Lupingyue** [reply]
> 其实大家也不会直接调用智能合约。智能合约有相应的前端页面，前端页面就是各种玩法的说明和入口。只要商业模式成立，都可以做的

好的，感谢科普🙏。我开了一个新帖子 #36 ，有什么新想法可以在新帖子讨论
